### PR TITLE
Fixed typo in README

### PR DESCRIPTION
**Pull request by eddiejaoude**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,4 @@ Describe here which problem you are working on during the event, and what your s
 
 ## Install or run this project
 
-Paste here a link to the project if it is a running website, or some indications on how to use your code for other github user who might be interested. The jury will take a look at what is in this repository during deliberations.
-
-
+Paste here a link to the project if it is a running website, or some indications on how to use your code for other GitHub user who might be interested. The jury will take a look at what is in this repository during deliberations.
```
Comment (steverabino) [inline README.md]: user -> users

**Review by steverabino — APPROVED** [review]
Suggestion added for one other missed typo

File: README.md (hunk 1 above)
Comment (eddiejaoude) [reply README.md]: Thank you @stevesmith2609 👍

**Comment by eddiejaoude** [comment]
Fixed that extra typo, thanks @stevesmith2609 👍

**Pull request merged**